ci/kokoro/install/Dockerfile.opensuse-leap

```diff
@@ -66,6 +75,16 @@ RUN cmake --build cmake-out --target install -- -j ${NCPU:-4}
 RUN ldconfig
 # ```
 
+# The following steps will install libraries and tools in `/usr/local`.
+# By default openSUSE/Leap does not search for shared libraries in these
+# directories, there are multiple ways to solve this problem, the following
+# steps are one solution:
+
+# ```bash
+RUN (echo "/usr/local/lib" ; "/usr/local/lib64") | \
```

Comment:
Here too, looks like it's missing an `echo`.

Reply:
Argh. Fixed.